src/Workspaces/Core/MSBuild/MSBuild/MSBuildWorkspace.cs

```diff
@@ -104,7 +102,7 @@ public static MSBuildWorkspace Create(IDictionary<string, string> properties, Ho
         /// <summary>
         /// Diagnostics logged while opening solutions, projects and documents.
         /// </summary>
-        public ImmutableList<WorkspaceDiagnostic> Diagnostics => _reporter.Diagnostics;
+        public ImmutableList<WorkspaceDiagnostic> Diagnostics => _reporter._diagnostics;
```

Comment:
i triply hate ths now.  having another type access an `_` member of another varaible looks like it's somehow accessing a privat.e